### DISCUSSION TITLE
Clip to recording

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#5775] [Ideas]
Don't remember & cannot find this idea...

When watching clips it seems that (at least for me) comes to situation that I'd like to see more about before or after recorded event.

After that, have to put in my mind date&time -> goes to the recordings and open that time.

It would be really nice if at clips there would be a direct link at this certain point from the recording.

**Comment by NickM-27** [comment]
Most likely you're referring to https://github.com/blakeblackshear/frigate/issues/2046

**Reply by McPaHa** [reply]
Yeap sorry... double it... my bad!
